Tests/UnitTests.WebView.WinForms/FunctionalTests/Navigation/NavigateTests.cs

```diff
@@ -52,18 +52,20 @@ public void NavigationFailedWithArgumentException()
         {
             WebView.Navigate(new Uri("/someresource", UriKind.Relative));
         }
-    }
-
+    }
+
+    // TestContext.TestRunResultsDirectory is only supported on .Net Framework
+#if NET462
```

Comment:
Need an alternative test for navigating to local file path for .NET Core 3.0

Reply:
That test is already [Ignored].